crates/core/tedge/src/cli/connect/command.rs

```diff
@@ -76,7 +76,7 @@ impl Command for ConnectCommand {
                 return match self.check_connection(&config) {
                     Ok(_) => {
                         let cloud = br_config.cloud_name.clone();
-                        println!("Connection check to {} cloud is successfull.\n", cloud);
+                        println!("Connection check to {} cloud is successful.\n", cloud);
```

Comment:
The same misspelling in line 134 and 239. Can you also fix them?

Line 134
```
println!("Connection check is successfull.\n");
```

Line 239
```
// If successfull in getting the jwt token '71,xxxxx', the connection is established.
```

Reply:
fixed